renderdocui/Windows/EventBrowser.cs

```diff
@@ -363,7 +363,17 @@ private void SetDrawcallTimes(TreelistView.Node n, Dictionary<uint, List<Counter
                 uint eid = GetNodeEventID(n);
 
                 if (times.ContainsKey(eid))
-                    duration = times[eid][0].value.d;
+                {
+                    duration = -1.0;
+                    foreach (var counter in times[eid])
+                    {
+                        if (counter.counterID == (uint) GPUCounters.EventGPUDuration)
+                        {
+                            duration = times[eid][0].value.d;
+                            break;
+                        }
+                    }
+                }
```

Comment:
I guess the changes in this file were for local testing,. so can be reverted? Since fetching more counters just gets wasted as we only look for one of them here. The FetchCounters etc calls are still available from python as you mention.